src/views/PositionPlot/PositionPlotView.tsx

```diff
@@ -52,35 +53,58 @@ const PositionPlotView: FunctionComponent<Props> = ({data, width, height}) => {
         context.stroke()
         context.setLineDash([]);
 
-        for (let dim of props.dimensions) {
-            context.strokeStyle = colorForUnitId(dim.dimensionIndex)
-            context.beginPath()
-            let first = true
-            for (let i=0; i<dim.pixelTimes.length; i++) {
-                const x = dim.pixelTimes[i]
-                const y = dim.pixelValues[i]
-                if (first) context.moveTo(x, y)
-                else context.lineTo(x, y)
-                first = false
-            }
-            context.stroke()
+        if (props.plotType === 'line' || !props.plotType) {
+            props.dimensions.forEach(dim => {
+                context.strokeStyle = colorForUnitId(dim.dimensionIndex)
+                context.lineWidth = 1.1 // hack--but fixes the 'disappearing lines' issue
+                context.beginPath()
+                dim.pixelTimes.forEach((x, ii) => {
+                    const y = dim.pixelValues[ii]
+                    ii === 0 ? context.moveTo(x, y) : context.lineTo(x, y)
+                })
+                context.stroke()
+            })
+        } else if (props.plotType === 'scatter') {
+            const circleWidthPx = 0.1 // could go even lighter
+            props.dimensions.forEach(d => {
+                context.fillStyle = colorForUnitId(d.dimensionIndex)
```

Comment:
Side thought--we could probably do away with the need for the `dimensionIndex` member altogether if we just take the index of `props.dimensions`. Not sure if there's any other significance to having the dimension index explicitly recorded in the data structure.